src/js/config/defaults.js

```diff
@@ -365,6 +380,14 @@ const defaults = {
             enabled: 'plyr--captions-enabled',
             active: 'plyr--captions-active',
         },
+        trim: {
+            enabled: 'plyr--trim-enabled',
+            active: 'plyr--trim-active',
+            // Trim tool
+            trimTool: 'plyr__trim-tool',
```

Comment:
what's the difference between the `--` and `__` ?

Reply:
So in the rest of the code base controls use -- syntax whereas features follow the __ I agree through me off

I have also amended the other comments made